contrib/ci/generate_dockerfile.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python3
+import os
+import sys
+
+directory = os.path.dirname(sys.argv[0])
```

Comment:
maybe checking len(argv)?

Reply:
Even when you're always grabbing 0? to get the path?  That should always be there.

Reply:
Yup, you're right, sorry.